extensions/amp-story/0.1/amp-story.js

```diff
@@ -450,7 +454,13 @@ export class AmpStory extends AMP.BaseElement {
           });
         })
         .then(() => this.switchTo_(firstPageEl.id))
-        .then(() => this.preloadPagesByDistance_());
+        .then(() => this.preloadPagesByDistance_())
+        .then(() => {
```

Comment:
a `this.setupParallax_()` that contains the few lines below would feel nicer given the current flow of the code here.